### PR TITLE
adds uniq as alias for union in README

### PR DESCRIPTION
**Pull request by zigahertz**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -322,7 +322,8 @@ Creates an array of unique values that are included in all given arrays.
 
 **[⬆️ back to top](#available-helpers)**
 
-#### `union`
+#### `union` _alias_ `uniq`
```
Comment (poteto) [inline README.md]: Could you move the `_alias_ uniq` to the next line? This breaks the TOC link :(

**Comment by zigahertz** [comment]
@poteto sorry about that! I updated the description, please let me know what you think. Alternatively - could make an entry for `uniq`, say that it is an alias for `union`, and have it link to the section on union, this way `uniq` would show up in the TOC.

**Comment by poteto** [comment]
This is great for now. Thanks!

**Comment by poteto** [comment]
@homu r+

**Pull request merged**